### PR TITLE
Small corrections from last merge to master

### PR DESCRIPTION
**Pull request by kathyoleary**
:green_apple:  Ready to go!  Several minor fixes from last merge to master.

### CODE REVIEW

**Comment by kathyoleary** [comment]
https://github.com/LMHT/LMHT.github.io/issues/187

File: elements/header.html [hunk 1]
```diff
@@ -40,9 +40,9 @@
     <code>
       <strong>&lt;header&gt;</strong>
         &lt;h1&gt;Title of the content&lt;/h1&gt;
-          &lt;div class="example"&gt;
+        &lt;div class="example"&gt;
             &lt;p&gt;This is a paragraph example.&lt;/p&gt;
```
Comment (rylnd) [inline elements/header.html]: If the parent element was outdented, shouldn't its children be as well?

**Comment by kathyoleary** [comment]
Yep, thought I had.  Just updated the fix.

**Comment by rylnd** [comment]
:+1:

**Comment by LipGlossary** [comment]
:+1:

**Pull request merged**